### ISSUE TITLE
Crash when casting with full package + class name

### ISSUE THREAD

**Issue by Graham42** [thorbenvh8/prettier-java#55]
Sample code:
```java
public class MyTest {
    public void testMethod(Throwable cause) {
        if (cause instanceof java.lang.Exception) {
            Exception ex = (java.lang.Exception)cause;
        }
    }
}
```
Stacktrace
```
[error] Unable to expand glob patterns: MyTest.java !**/node_modules/** !./node_modules/**
[error] Cannot read property 'stack' of undefined
```

**Comment by Graham42** [comment]
Debugging this a bit, found that when chevrotain-java throws `MismatchedTokenException` at https://github.com/thorbenvh8/chevrotain-java/blob/629a78af91c91f1a6ddd9f05b7d98afff3268d9d/src/visitor.js#L2920
... upstream prettier dies because the error object does not have a `stack` property. Should I open an upstream bug for this?

**Comment by thorbenvh8** [comment]
Great debugging! Fixed in next release!

**Comment by thorbenvh8** [comment]
Fixed in 0.0.10!
Thanks for the raising the issue! 👍
